### ISSUE TITLE
MSnbase fails to install on Ubuntu 16.04.3

### ISSUE THREAD

**Issue by prvst** [lgatto/MSnbase#245]
Hi @lgatto ;

MSnbase currently fails to install on latest R and Bioconductor because of  the mzR dependency on libboost-thread. 

The issue is referenced  [here](https://github.com/sneumann/mzR/issues/108), on the mzR repository, and by what they mention there, it could be just related to the latest Ubuntu LTS (16.04)

The error was already reported, so I'm just posting the issue here as a reference for others.

```
/usr/lib/R/etc/Makeconf:143: recipe for target 'boost/thread/src/pthread/thread.o' failed
make: *** [boost/thread/src/pthread/thread.o] Error 1
ERROR: compilation failed for package ‘mzR’
* removing ‘/home/felipevl/R/x86_64-pc-linux-gnu-library/3.2/mzR’
ERROR: dependency ‘mzR’ is not available for package ‘MSnbase’
* removing ‘/home/felipevl/R/x86_64-pc-linux-gnu-library/3.2/MSnbase’
```

**Comment by lgatto** [comment]
Thank you for the report. I'll try to get my hands on an Ubuntu 16.04 or later (I'm still running 14.04) and explore the issue.

**Comment by prvst** [comment]
some updates here; the problem can be fixed by updating the R version to 3.4.1. More comments can be found on the mzR issue posted above.
I think we can close this.

Thanks

**Comment by lgatto** [comment]
Yes, I follow all issues on `mzR`, but thank you for following up here. I'm closing the issue, but feel free to reopen it if needed.
